pkgs/by-name/fr/freecad/package.nix

```diff
@@ -40,7 +40,7 @@
   nix-update-script,
 }:
 let
-  inherit (python311Packages)
+  inherit (python3Packages)
```

Comment:
`shiboken2`

Reply:
For freecad, the correct change would have been https://github.com/NixOS/nixpkgs/pull/408577/commits/2485c747cc444bb13e98b004b0fac22244e61ee2, also described in https://github.com/NixOS/nixpkgs/pull/408577#pullrequestreview-2851866694